package.json

```diff
@@ -46,8 +46,9 @@
     "format": "prettier --write '**/*.{ts,tsx,js}'",
     "format-check": "prettier --check '**/*.{ts,tsx,js}'",
     "jest": "jest",
-    "snapshot": "jest --updateSnapshot",
+    "jestUpdate": "jest --updateSnapshot",
```

Comment:
`jest-update` would have been a more consistent name (similar to `format-check`, `build-ci`, etc.).

Also, IMHO, `update-snapshot` as a name, more accurately reflects what the script does.

Reply:
+1 for `update-snapshot`

Reply:
My bad I was hoping for that to be in a separate PR and ended up committing to this one. 

On the naming conventions, normally in JavaScript we use attached camelCase instead of the kebab-case. So, I'd argue that we might need to change the other scripts to have that camelCase in a new PR. Unless there's an issue?

Reply:
For JS code, `camelCase` is the defacto convention, whereas for npm scripts, kebab-case seems to be more preferred across the community.

The [`npm` CLI](https://github.com/npm/cli/blob/latest/package.json#L292) uses `kebab-case` for naming scripts. Many of the top [most-depended-upon](https://www.npmjs.com/browse/depended) packages on `npm` use `kebab-case`, with the notable exception of `lodash`, which uses colons.

Reply:
Raised a PR with the above change: https://github.com/nodejs/nodejs.dev/pull/230. PTAL.